pom.xml

```diff
@@ -1929,6 +1943,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
+            <version>${plugin.surefire.version}</version>
```

Comment:
The version is already stored here, so this is unnecessary.
https://github.com/apache/zeppelin/blob/aa4731e69916ed20a3aad56a89544d7a08d57d24/pom.xml#L1605-L1609